src/App.js

```diff
@@ -1,13 +1,84 @@
+import { Routes, Route, Link } from 'react-router-dom';
 import './App.css';
 import Calculator from './components/Calculator';
 import './components/Calculator.css';
 
 function App() {
   return (
-    <div className="App">
-      <Calculator />
+    <div>
+      <div className="top-bar">
+        <h1>Math Magicians</h1>
+        <nav className="links">
+          <Link to="/">Home</Link>
+          <Link to="/Calculator">Calculator</Link>
+          <Link to="/quote">Quote</Link>
+        </nav>
```

Comment:
- [ ] [OPTIONAL] It would be better if you separate this piece of code into Navbar `component` file that will make the app looks clean and